docs/hardware.md

```diff
@@ -23,13 +23,9 @@ DietPi runs also on one of the first models **Raspberry Pi 1**, launched in 2012
 
 ![Raspberry Pi 1 Model B photo](assets/images/raspberry-pi-1b.jpg){: width="500" height="242" loading="lazy"}
 
-and even on the compact form **Raspberry Pi Zero W**
+and even on the compact form **Raspberry Pi Zero**, both **Zero W / WH** and **Zero 2 W**
 
-![Raspberry Pi Zero W photo](assets/images/raspberry-pi-zero-w.jpg){: width="500" height="333" loading="lazy"}
```

Comment:
Don't forget to delete the image, when it's not used anymore.

Reply:
done